src/rmp/src/Restructure.cpp

```diff
@@ -66,6 +71,37 @@ using utl::RMP;
 
 namespace rmp {
 
+////////////////////////////////////////////////////////////////////////
+///                        DECLARATIONS                              ///
+////////////////////////////////////////////////////////////////////////
+
+#if defined(ABC_NAMESPACE)
+namespace ABC_NAMESPACE
+{
+#elif defined(__cplusplus)
+extern "C"
+{
+#endif
+
+// procedures to start and stop the ABC framework
+// (should be called before and after the ABC procedures are called)
+void   Abc_Start();
+void   Abc_Stop();
```

Comment:
Why are you declaring these rather than including base/main/abcapis.h.

Reply:
That is how the abc demo.c is suggesting to do it. Let me try this and @QuantamHD 's suggestion to see which works best to avoid the need for the forward declarations.

Reply:
I went with the #include abcapis.h approach.

Reply:
You should do both. That way ABC doesn't encroach on openroad's global namespace.